charts/grnoc-telegraf/templates/deployment.yaml

```diff
@@ -43,6 +43,9 @@ spec:
                 secretKeyRef:
                   name: {{ .Values.grnocOutput.passwordSecretName }}
                   key: password
+          resources:
+            requests: 
+              memory: '{{.Values.requestMemory}}'
```

Comment:
Just a nit, but can you add spaces before and after  the braces, e.g. like the entry above.  I don't think it'll cause template errors but it looks better and might prevent an error

Reply:
added the spaces; will keep this in mind for the future